### DISCUSSION TITLE
What is the 'Main' choice in Permission?

### DISCUSSION THREAD

**Discussion by willardinteractive** [orchidsoftware/platform#1697] [Q&A] [answered]
When I look at the permissions settings for a user there are two groups broken down like this:

Main
- Main

System
- Attachment
- Roles
- Users

What are the main group and selections in reference to? Which file(s) are these defined in? I only see the Systems group defined in app/Orchid/PlatformProvider.php

Thanks!

**Comment by tabuna** [comment, marked as answer]
Hi @willardinteractive, access to the panel.
This is checked in the [middleware](https://github.com/orchidsoftware/platform/blob/f6951d8fb68c5243f0a8e1c2a6cfaca3dc5497e0/config/platform.php#L52), so that it would not be enough just to log in, but also to have this permission.

**Reply by willardinteractive** [reply]
Thank you!

**Comment by StenSeegel** [comment]
Hey, I'm looking at this topic right now. It seems to me like the "main" permission for "plattform.index" is hardcoded and can't be changed in the PlatformProvider.php. The following code produces a duplicate of the platform.index permission and I then get one named "Admin Access" and another still named "Main" still placed in a group called "Main" 
```
ItemPermission::group(__('Admin Panel'))
                ->addPermission('platform.index', __('Admin Access')),
```

 Is there any other way to at rename the permission?

**Reply by tabuna** [reply]
It seems that one possible solution could be adding functionality for removing or replacing permissions. On the other hand, that might feel a bit too heavy. Maybe we could consider a PR that stores a more generic name instead of `Main`?
